gcloud/_helpers.py

```diff
@@ -584,29 +588,44 @@ def make_stub(credentials, user_agent, stub_factory, host, port):
     :type user_agent: str
     :param user_agent: (Optional) The user agent to be used with API requests.
 
-    :type stub_factory: callable
-    :param stub_factory: A factory which will create a gRPC stub for
-                         a given service.
+    :type stub_class: type
+    :param stub_class: A gRPC stub type for a given service.
 
     :type host: str
     :param host: The host for the service.
 
     :type port: int
     :param port: The port for the service.
 
-    :rtype: :class:`grpc.beta._stub._AutoIntermediary`
+    :rtype: object, instance of ``stub_class``
     :returns: The stub object used to make gRPC requests to a given API.
     """
     # Leaving the first argument to ssl_channel_credentials() as None
     # loads root certificates from `grpc/_adapter/credentials/roots.pem`.
-    transport_creds = implementations.ssl_channel_credentials(None, None, None)
+    transport_creds = grpc.ssl_channel_credentials(None, None, None)
     custom_metadata_plugin = MetadataPlugin(credentials, user_agent)
-    auth_creds = implementations.metadata_call_credentials(
+    auth_creds = grpc.metadata_call_credentials(
         custom_metadata_plugin, name='google_creds')
-    channel_creds = implementations.composite_channel_credentials(
+    channel_creds = grpc.composite_channel_credentials(
         transport_creds, auth_creds)
-    channel = implementations.secure_channel(host, port, channel_creds)
-    return stub_factory(channel)
+    target = '%s:%d' % (host, port)
+    channel = grpc.secure_channel(target, channel_creds)
+    return stub_class(channel)
+
+
+def exc_to_code(exc):
```

Comment:
... or is it the case that some other code that is called by the code I'm seeing in this change is still using the gRPC Python Beta API, so this needs to be able to handle exceptions from either?